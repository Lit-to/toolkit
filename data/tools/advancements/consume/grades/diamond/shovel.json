{
    "criteria": {
        "use_item": {
            "trigger": "minecraft:consume_item",
            "conditions": {
                "item": {
                    "items": [
                        "minecraft:diamond_shovel"
                    ]
                }
            }
        }
    },
    "rewards": {
        "function": "tools:from_adv/grades/diamond/shovel"
    }
}