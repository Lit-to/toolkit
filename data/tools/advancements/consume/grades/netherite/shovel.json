{
    "criteria": {
        "use_item": {
            "trigger": "minecraft:consume_item",
            "conditions": {
                "item": {
                    "items": [
                        "minecraft:netherite_shovel"
                    ]
                }
            }
        }
    },
    "rewards": {
        "function": "tools:from_adv/grades/netherite/shovel"
    }
}