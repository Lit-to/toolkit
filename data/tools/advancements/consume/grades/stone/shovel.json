{
    "criteria": {
        "use_item": {
            "trigger": "minecraft:consume_item",
            "conditions": {
                "item": {
                    "items": [
                        "minecraft:stone_shovel"
                    ]
                }
            }
        }
    },
    "rewards": {
        "function": "tools:from_adv/grades/stone/shovel"
    }
}