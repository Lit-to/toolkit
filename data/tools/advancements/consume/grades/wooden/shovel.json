{
    "criteria": {
        "use_item": {
            "trigger": "minecraft:consume_item",
            "conditions": {
                "item": {
                    "items": [
                        "minecraft:wooden_shovel"
                    ]
                }
            }
        }
    },
    "rewards": {
        "function": "tools:from_adv/grades/wooden/shovel"
    }
}