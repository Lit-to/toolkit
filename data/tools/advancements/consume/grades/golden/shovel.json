{
    "criteria": {
        "use_item": {
            "trigger": "minecraft:consume_item",
            "conditions": {
                "item": {
                    "items": [
                        "minecraft:golden_shovel"
                    ]
                }
            }
        }
    },
    "rewards": {
        "function": "tools:from_adv/grades/golden/shovel"
    }
}